tests/foreman/ui/test_adusergroups.py

```diff
@@ -157,3 +160,124 @@ def test_create_external_adusergroup_3(self):
         @Status: Manual
 
         """
+
+    @stubbed()
+    def test_delete_external_adUserGroup(self):
```

Comment:
This should be updated to `test_delete_external_adusergroup` or `test_delete_external_ad_user_group`. Also there are other methods that should not use CamelCase method names.